### ISSUE TITLE
TabView: Unable to perform drag operation on Tab items using touch Narrator

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#3528]
**Describe the bug**

**Steps to reproduce the bug**

Pre-requisite:
- Turn on Narrator.
- Set Narrator in Item Mode.
- Touch supported monitor required.

1. Launch XAML Controls Gallery application.
2. Navigate to "All Controls" button and activate it.
3. Navigate to "TabView" and activate it.
4. create at least 3-4 tabs.
5. Navigate to any tab and double tap to activate it.
6. Hold with one finger and then tap anywhere on the screen with two fingers
7. observe narrator is reading.

**Expected behavior**

User should be able to drag the tab item.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/97661664-37b18580-1a32-11eb-9aa5-5d62326b2be0.png)

**Version Info**

Xaml Controls Gallery version 1.2.16.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/23661800)
